adi_diagnostic_report

```diff
@@ -291,12 +291,15 @@ def get_sys_bus():
 @diagnostic('apt-package', "Installed Debian packages checker", True)
 def get_apt_info():
     file_path = "/var/lib/apt/lists/swdownloads.analog.com_cse_adi-repo_debian_dists_bookworm_main_binary-all_Packages"
-    grep_command = f"grep '^Package: ' {file_path} | awk '{{print $2}}'"
-    packages = subprocess.check_output(grep_command, shell=True).decode('utf-8').splitlines()
-    dpkg_command = f"dpkg -s {' '.join(packages)} 2>/dev/null | grep -B 1 -A 4 'ok installed'"
-    result = subprocess.check_output(dpkg_command, shell=True).decode('utf-8')
-    pattern = r"(Package|Architecture|Version):\s*(.+)"
-    matches = re.findall(pattern, result)
+    if os.path.isfile(file_path):
+        grep_command = f"grep '^Package: ' {file_path} | awk '{{print $2}}'"
+        packages = subprocess.check_output(grep_command, shell=True).decode('utf-8').splitlines()
+        dpkg_command = f"dpkg -s {' '.join(packages)} 2>/dev/null | grep -B 1 -A 4 'ok installed'"
+        result = subprocess.check_output(dpkg_command, shell=True).decode('utf-8')
+        pattern = r"(Package|Architecture|Version):\s*(.+)"
+        matches = re.findall(pattern, result)
+    else:
+        return {'apt-pack': "The file path doesn't exist."}
```

Comment:
Can you also print the value of `file_path` here?

Reply:
done